esignet-countryB.properties

```diff
@@ -0,0 +1,500 @@
+#mosip.kernel.keymanager.hsm.keystore-type=PKCS12
+#mosip.kernel.keymanager.hsm.keystore-pass=${softhsm.idp.pin}
+
+#Type of keystore, Supported Types: PKCS11, PKCS12, Offline, JCE
+mosip.kernel.keymanager.hsm.keystore-type=PKCS11
+# For PKCS11 provide Path of config file.
+# For PKCS12 keystore type provide the p12/pfx file path. P12 file will be created internally so provide only file path & file name.
+# For Offline & JCE property can be left blank, specified value will be ignored.
+mosip.kernel.keymanager.hsm.config-path=/config/softhsm-application.conf
+# Passkey of keystore for PKCS11, PKCS12
+# For Offline & JCE proer can be left blank. JCE password use other JCE specific properties.
+mosip.kernel.keymanager.hsm.keystore-pass=${softhsm.esignet.security.pin}
+
+
+mosip.kernel.keymanager.certificate.default.common-name=www.mosip.io
+mosip.kernel.keymanager.certificate.default.organizational-unit=MOSIP-TECH-CENTER
+mosip.kernel.keymanager.certificate.default.organization=IITB
+mosip.kernel.keymanager.certificate.default.location=BANGALORE
+mosip.kernel.keymanager.certificate.default.state=KA
+mosip.kernel.keymanager.certificate.default.country=IN
+
+mosip.kernel.keymanager.softhsm.certificate.common-name=www.mosip.io
+mosip.kernel.keymanager.softhsm.certificate.organizational-unit=MOSIP
+mosip.kernel.keymanager.softhsm.certificate.organization=IITB
+mosip.kernel.keymanager.softhsm.certificate.country=IN
+
+# Application Id for PMS master key.
+mosip.kernel.partner.sign.masterkey.application.id=PMS
+mosip.kernel.partner.allowed.domains=DEVICE
+
+mosip.kernel.keymanager-service-validate-url=https://${mosip.hostname}/keymanager/validate
+mosip.kernel.keymanager.jwtsign.validate.json=false
+mosip.keymanager.dao.enabled=false
+crypto.PrependThumbprint.enable=true
+
+mosip.kernel.keymgr.hsm.health.check.enabled=true
+mosip.kernel.keymgr.hsm.health.key.app-id=OIDC_SERVICE
+mosip.kernel.keymgr.hsm.healthkey.ref-id=TRANSACTION_CACHE
+mosip.kernel.keymgr.hsm.health.check.encrypt=true
+
+## -------------------------------------------- Cross-border-auth config ----------------------------------------------------------
+mosip.esignet.proxy.relying-party-service-url=http://proxy-mock-relying-party-service.esignet/fetchUserInfo
+mosip.esignet.proxy.oidc-client-id=ion1rzK_vIr8387gGD35MOSO6bq3KoiVCbM0oa68C4c
+mosip.esignet.proxy.oidc-client-redirect-uri=https://esignet.dev1.mosip.net/proxy-authorization
+
+## -------------------------------------------- IDP-UI config ----------------------------------------------------------
+# NOTE:
+# 1. linked-transaction-expire-in-secs value should be a sum of mosip.esignet.authentication-expire-in-secs and linked cache expire in seconds under mosip.esignet.cache.expire-in-seconds property
+# 2. A new Qrcode will be autogenerated before the expiry of current qr-code, and the time difference in seconds for the same is defined in wallet.qr-code-buffer-in-secs property
+
+mosip.esignet.ui.cross-border-auth.config={{\

```

Comment:
i think this will not be required here. Can you pls confirm @anshulv1401

Reply:
@ase-101 @Rakshithb1 , I'll raise a PR with the changes for countryA nd countryB